pangeo-notebook/environment.yml

```
@@ -65,6 +66,7 @@ dependencies:
  - jupyter-resource-usage
  - jupyter-sshd-proxy
  - kerchunk
+ - libgdal-hdf5
```

_Originally posted by @wildintellect in [#36](https://github.com/NASA-IMPACT/pangeo-notebook-veda-image/issues/36#issuecomment-3137683085)_
> 
> Open question what about netcdf support in gdal? 

@wildintellect, do you want `libgdal-netcdf` also? Since [`libgdal-netcdf` depends on `libgdal-hdf5` and `libgdal-hdf4`](https://github.com/conda-forge/gdal-feedstock/blob/891251423f09b5ae41c7acedebc82fd386648e75/recipe/meta.yaml#L464-L467), we could switch this line to:

```suggestion
 - libgdal-netcdf
```

---

Ok, I've applied the change